lua/lspsaga/diagnostic.lua

```diff
@@ -107,7 +107,7 @@ local show_diagnostics = function(opts, get_diagnostics)
     table.insert(highlights, 2, { 0, 'LspSagaDiagnosticTruncateLine' })
   end
 
-  local content_opts = { contents = lines, filetype = "plaintext", highlight = "LspSagaDiagnosticBorder" }
+  local content_opts = { contents = lines, filetype = "LspsagaDiagnostic", highlight = "LspSagaDiagnosticBorder" }
```

Comment:
hmm I think this only relevant for the border and not the diagnostic window it self, no?